### PR TITLE
Angular: Remove dead code in client

### PR DESCRIPTION
**Pull request by ThibaudAV**
Issue: n/a

## What I did

Remove dead code in client. This code is now longer useful with new RendererFactory

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

### CODE REVIEW

File: app/angular/src/renderer.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 export { computesTemplateSourceFromComponent } from './client/preview/angular-beta/ComputesTemplateFromComponent';
-export { RendererService } from './client/preview/angular-beta/RendererService';
```
Comment (ThibaudAV) [inline app/angular/src/renderer.ts:2]: Previously used by docs addon. but now docs use RendererFactory witch return dedicated DocsRenderer instance

File: app/angular/src/client/preview/angular-beta/RendererService.test.ts [hunk 2]
```diff
@@ -1,241 +0,0 @@
...
-        component: FooComponent,
-        parameters: {},
-      });
-
-      expect(document.body.getElementsByTagName('storybook-wrapper')[0].innerHTML).toBe(
-        '<foo>🦊</foo><!--container-->'
-      );
-    });
-
-    it('should handle circular reference in moduleMetadata', async () => {
```
Comment (Marklb) [inline app/angular/src/client/preview/angular-beta/RendererService.test.ts:63]: This test should probably be moved to `RendererFactory.test.ts`.

**Review by shilman — APPROVED** [review]
LGTM!! 🧹🧹🧹

File: app/angular/src/client/preview/angular-beta/RendererService.test.ts (hunk 2 above)
Comment (ThibaudAV) [reply app/angular/src/client/preview/angular-beta/RendererService.test.ts:63]: Yes ! Ty

Comment (ThibaudAV) [reply app/angular/src/client/preview/angular-beta/RendererService.test.ts:63]: Done

**Pull request merged**